font-types/src/name_id.rs

```diff
@@ -0,0 +1,184 @@
+use core::fmt;
+
+/// Identifier for an informational string (or name).
+///
+/// A set of pre-defined identifiers exist for accessing names and other various metadata
```

Comment:
```suggestion
/// A set of predefined identifiers exist for accessing names and other various metadata
```

not sure if this is a style thing? I think predefined is a word.